index.js

```diff
@@ -0,0 +1,31 @@
+// Функция принимает 2 массива.
+// Возвращает новый массив, который состоит только из тех элементов,
+// которые встретились в одном массиве, но не встретились в другом
+
+//O(2n^2)
+function arrayDiff(array1, array2) {
+  return array1
+    .filter((el) => !array2.includes(el))
+    .concat(array2.filter((el) => !array1.includes(el)));
+}
+
+//O(4n)
+function arrayDiff(array1, array2) {
```

Comment:
pretty much good solution, but we can use common object intead of Map and build it only ones

Reply:
changed to Set